composer.json

```diff
@@ -17,11 +17,13 @@
     "require": {
         "php":                           ">=5.4",
         "behat/mink-browserkit-driver":  "~1.2@dev",
-        "fabpot/goutte":                 "~1.0.4|~2.0|~3.1"
+        "fabpot/goutte":                 "~1.0.4|~2.0|~3.1|^4.0"
     },
 
     "require-dev": {
-        "mink/driver-testsuite": "dev-master"
+        "mink/driver-testsuite": "dev-master",
+        "phpunit/phpunit": "^9.0",
```

Comment:
Please remove, because PHPUnit is already indirectly required via `mink/driver-testsuite`. Locking to PHPUnit 9.x would also break builds for PHP versions not supported by the PHPUnit 9.x.